src/constants/device-constants.ts

```diff
@@ -1,7 +1,9 @@
 export const PHONE_TYPES = Object.freeze({
   IPHONE: "iphone",
   ANDROID: "android",
-});
+} as const);
+
+export type PhoneType = (typeof PHONE_TYPES)[keyof typeof PHONE_TYPES];
```

Comment:
We should be able to use a Typescript string enum for this (which can also be used as a type) ?
https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

Though we'd probably want to apply the same to all constants so happy for this to be a separate tech debt ticket